### PR TITLE
 [ch3188] Redirect old registration to adhesion with 301

### PR DESCRIPTION
**Pull request by cacahouwete**
https://app.clubhouse.io/enmarche/story/3188/transf%C3%A9rer-le-contenu-de-inscription-vers-inscription-utilisateur-pour-permettre-ensuite-une-redirection-301-de-inscription

### CODE REVIEW

File: src/Controller/EnMarche/MembershipController.php [hunk 1]
```diff
@@ -20,16 +20,28 @@
...
 
 class MembershipController extends Controller
 {
+    /**
+     * Redirection for SEO
+     *
+     * @Route("/inscription", name="app_membership_redirect")
+     * @Method("GET|POST")
+     */
+    public function registerRedirectToAdhesionAction(Request $request): RedirectResponse
```
Comment (B-Galati) [inline src/Controller/EnMarche/MembershipController.php]: Use `\AppBundle\Redirection\LegacyRedirectionsSubscriber` instead ;-)

File: src/Controller/EnMarche/MembershipController.php [hunk 2]
```diff
@@ -20,16 +20,28 @@
...
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Security\Core\Authorization\AuthorizationChecker;
 
 class MembershipController extends Controller
 {
+    /**
+     * Redirection for SEO
+     *
+     * @Route("/inscription", name="app_membership_redirect")
```
Comment (ottaviano) [inline src/Controller/EnMarche/MembershipController.php]: http://symfony.com/doc/current/routing/redirect_in_config.html

File: src/Controller/EnMarche/MembershipController.php (hunk 1 above)
Comment (B-Galati) [reply src/Controller/EnMarche/MembershipController.php]: The query string part should be redirected too.

Comment (cacahouwete) [reply src/Controller/EnMarche/MembershipController.php]: ok

**Pull request merged**